test/selenium/lib/view/signed_out/accessors.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+var webdriver = require('selenium-webdriver');
+var By = require('selenium-webdriver').By;
+
+var SELECTORS = Object.freeze({
+    page: By.id('signin')
+});
+
+function SignedOutAccessor(driver) {
+  this.driver = driver;
+};
+
+SignedOutAccessor.prototype = {
+   get isSignOutPage(){
+   	  return this.driver.isElementPresent(SELECTORS.page);
```

Comment:
I got an error I do not understand...If I do:
return this.driver.wait(webdriver.until.elementIsVisible(SELECTORS.page));
   }
the error is: TypeError: element.isDisplayed is not a function
BUT, if I do this: 
get root() {
   	var page = this.driver.findElement(webdriver.By.id('signin'));
  	return this.driver.wait(webdriver.until.elementIsVisible(page));
   }
It works and the test passes...

Should I define the selector inside the root method or look for another solution?